extract_strings.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Let's turn this into an `invoke` task instead, so that it works on every platform. If you look at tasks.py, you can see that most commands are instructions to either run "other, smaller tasks", or for small tasks are simply execution instructions for manage.py, copying a dir, etc.